githubbot/githubbot/db.go

```diff
@@ -163,3 +162,41 @@ func (d *DB) DeleteToken(identifier string) error {
 	})
 	return err
 }
+
+// preferences
+
+type UserPreferences struct {
+	Mention bool
+}
+
+func (d *DB) GetUserPreferences(username string) (prefs *UserPreferences, err error) {
+	row := d.DB.QueryRow(`SELECT mention
+		FROM user_prefs
+		WHERE username = ?`, username)
+	prefs = &UserPreferences{}
```

Comment:
naming the return var defines the variable, can nuke this line!

Reply:
@joshblum was getting nil pointer errors without this line, not sure how to resolve that